packages/styled-components/components/searchForm/index.js

```diff
@@ -57,6 +57,7 @@ const SearchForm = (props) => {
         ref={register}
         type="text"
       />
+      {/* @TODO Add GTM event.  */}
```

Comment:
Did you mean to do this before this was merged?

Reply:
@joemcgill No, but I'll make a new ticket so it doesn't get lost in the shuffle.